packages/records/lib/models/records.integration.test.ts

```diff
@@ -13,13 +13,13 @@ describe('Records service', () => {
         await migrate();
     });
 
-    afterEach(async () => {
+    afterAll(async () => {
         await db(RECORDS_TABLE).truncate();
     });
 
     it('Should write records', async () => {
-        const connectionId = 1;
-        const environmentId = 2;
+        const connectionId = Math.floor(Math.random() * 1000000);
```

Comment:
you can use nanoid (from nango utils) instead for clarity

Reply:
I don't think we can - nanoid generates strings and these are both integers.

Reply:
ah yes didn't realised that 👍🏻